tox.ini

```diff
@@ -13,12 +13,13 @@ deps =
 
     dj111: Django>=1.11,<2.0
     dj20: Django>=2.0,<2.1
+    # Frontend tests fail after dj2.2.2 because of phantomjs and the change
```

Comment:
We will need @FinalAngel and @vxsx to help us decide on the best solution for this I think.

Reply:
To clarify; it works if you test things manually. It's just that phantomjs / casper is outdated.

Reply:
it fails most likely due to how clicks are implemented in casperjs/phantomjs, so short term it requires adaptions in the tests to account for the change in django, but best solution would be to move away from casperjs to something puppeteer based or smth like cypress

django cms would also need to do that eventually, but it's likely a much larger effort

i'm ok with disabling the faulty test for now

Reply:
Hi @vxsx I was never able to fully understand how you disable tests in casperjs. I mean you can comment out the test, but is there another way?

+1 for cypress.

Reply:
hi!
yeah there's no other way that i know of

Reply:
@jonathan-s You had mentioned that you wanted to use Cypress so I guess this is music. Can we disable or comment out the failing test to unblock our current issue and then look at rewriting to use Cypress? Can be an ongoing task if @vxsx agrees.

Reply:
@vxsx I have now disabled the offending test by commenting it out.

Reply:
👍